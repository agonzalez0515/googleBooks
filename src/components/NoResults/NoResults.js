import React from 'react'
import noResults from '../../assets/noResults.svg'

const NoResults = () => {
  return (  
    <div>
      <img src={noResults} className='placeholderImg' alt='Empty clipboards' />
      <p style={{textAlign: 'center', fontSize: '1.3rem'}}>No books found. Try again!</p>
    </div>
  )
}
 
export default NoResults
