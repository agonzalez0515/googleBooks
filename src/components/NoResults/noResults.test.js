import React from 'react'
import { render } from '@testing-library/react'
import NoResults from './NoResults'

it('renders', () => {
  const { asFragment }= render(<NoResults />)
  expect(asFragment()).toMatchSnapshot()
})
  
