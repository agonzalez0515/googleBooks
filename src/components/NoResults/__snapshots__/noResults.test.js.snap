// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="Empty clipboards"
      class="placeholderImg"
      src="noResults.svg"
    />
    <p
      style="text-align: center; font-size: 1.3rem;"
    >
      No books found. Try again!
    </p>
  </div>
</DocumentFragment>
`;
